README.md

```diff
@@ -64,7 +64,16 @@ func testOutlets() {
 }
 ```
 
-#### Testing constraints
+### Testing UIKit
+#### Testing Individual Components
+##### UIControl
+
+```swift
```

Comment:
Hmm, all of the other sections have a plain English description.  Even if this is pretty straight forward, can we add one here?  Then I think everything else looks good.